boutiques_descriptors/deepprep_24_1_2.json

```diff
@@ -44,7 +44,129 @@
             "name": "fs_license_file",
             "optional": false,
             "type": "File",
-            "value-key": "<fs_license_file>"
+            "value-key": "[fs_license_file]"
+        },
+        {
+            "description": "A space delimited list of participant identifiers or a single identifier (the sub- prefix can be removed)",
+            "id": "participant_label",
+            "name": "participant_label",
+            "optional": true,
+            "type": "String",
+            "value-key": "[--participant_label '001 002 003']"
+        },
+        {
+            "description": "The output directory of Recon files, default is <output_dir>/Recon",
+            "id": "subjects_dir",
+            "name": "subjects_dir",
+            "optional": true,
+            "type": "File",
+            "value-key": "[--subjects_dir PATH]"
+        },
+        {
+            "description": "Only the anatomical images will be processed.",
+            "id": "anat_only",
+            "name": "anat_only",
+            "optional": true,
+            "type": "Flag",
+            "command-line-flag": "--anat_only",
+            "value-key": "[--anat_only]"
+        },
+        {
+            "description": "Only the functional images will be processed, where Recon files are pre-requested.",
+            "id": "bold_only",
+            "name": "bold_only",
+            "optional": true,
+            "type": "Flag",
+            "command-line-flag": "--bold_only",
+            "value-key": "[--bold_only]"
+        },
+        {
+            "description": "Applies susceptibility distortion correction (SDC).",
+            "id": "bold_sdc",
+            "name": "bold_sdc",
+            "optional": true,
+            "type": "Flag",
+            "command-line-flag": "--bold_sdc",
+            "value-key": "[--bold_sdc]",
+            "default-value": true
+        },
+        {
+            "description": "Generates confounds derived from BOLD fMRI, such as head motion variables and global signals.",
+            "id": "bold_confounds",
+            "name": "bold_confounds",
+            "optional": true,
+            "type": "Flag",
+            "command-line-flag": "--bold_confounds",
+            "value-key": "[--bold_confounds]",
+            "default-value": true
+        },
+        {
+            "description": "Skip n frames of BOLD fMRI",
+            "id": "bold_skip_frames",
+            "name": "bold_skip_frames",
+            "optional": true,
+            "type": "Number",
+            "value-key": "[--bold_skip_frame 0]",
+            "default-value": 0
+        },
```

Comment:
I am very surprised that you were able to launch it, did you test all the new options? You might forgot the command-line-flag here and few other places.